src/utils/config-helper.ts

```diff
@@ -7,6 +7,9 @@ export default {
    * fetch config from environment and API
    */
   fetchConfig () {
+    const { name, version } = require('../../package.json')
```

Comment:
Does this work in build time?
Something like JSON.parse(unescape(process.env.PACKAGE_JSON || '%7Bversion%3A0%7D')).version
there is also some concern about loading all of the package.json
you could add something like this to vue.config.js
const fs = require('fs')
const packageJson = fs.readFileSync('./package.json')
const version = JSON.parse(packageJson).version || 0
module.exports = {
    configureWebpack: {
        plugins: [
            new webpack.DefinePlugin({
                'process.env': {
                    PACKAGE_VERSION: '"' + version + '"'
                }
            })
        ]
    }
}
Which doesn't expose the package.json in the built applicatioin

Reply:
As discussed, I have removed the subject code and we can revisit this in /bcgov/entity#2555.